lib/For.js

```diff
@@ -10,5 +10,6 @@ export default For = (initialization, condition, updation, fn) => {
   if(condition(initialization)) {
     fn(initialization)
     For(updation(initialization), condition, updation, fn)
-  } else return
```

Comment:
I'm personally a big fan of this design pattern -- removing an else statement. Truth be told, `else` is an archaic keyword, a relic of the early programming days that has stealthily wriggled its way into the roughly seventy-word vernacular of nearly all of our contemporaries. Programmers back then had no idea what they were doing because computers were still very much in their infancy 👶 at this time. Thank you for this contribution, I'll be merging this to our main branch posthaste!